### PR TITLE
Update Get-SmbShare.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated Example 2 and 3

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1923

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/smbshare/Get-SmbShare.md [hunk 1]
```diff
@@ -69,15 +69,15 @@ VMS5                          *                             D:\VMS
...
+### Example 2: Get a specific SMB share on a the local computer
 ```
 PS C:\>Get-SmbShare -Name "VMS1"
 Name                          ScopeName                     Path                          Description 
 ----                          ---------                     ----                          ----------- 
 VMS1                          Contoso-FS                    I:\VMS
 ```
 
-This command retrieves the SMB shares on the computer named VMS1.
+This command retrieves information about the SMB share named 'VMS1' on the local computer
```
Comment (tseward) [inline docset/windows/smbshare/Get-SmbShare.md:80]: Add a period at the end of the sentence.

File: docset/winserver2012r2-ps/smbshare/Get-SmbShare.md [hunk 2]
```diff
@@ -66,33 +66,32 @@ VMS4                          Contoso-SO                    C:\ClusterStorage\Vo
...
 ```
-PS C:\>Get-SmbShare -Name VMS1
+PS C:\>Get-SmbShare -Name "VMS1"
 Name                          ScopeName                     Path                          Description 
 ----                          ---------                     ----                          ----------- 
 VMS1                          Contoso-FS                    I:\VMS
 ```
 
-This example retrieves the SMB shares on the computer named VMS1.
+This command retrieves information about the SMB share named 'VMS1' on the local computer
```
Comment (tseward) [inline docset/winserver2012r2-ps/smbshare/Get-SmbShare.md:79]: Add a period at the end of the sentence.

File: docset/windows/smbshare/Get-SmbShare.md [hunk 3]
```diff
@@ -69,15 +69,15 @@ VMS5                          *                             D:\VMS
 
 This command retrieves the SMB shares on the computer.
 
-### Example 2: Get a specific SMB share on a specific computer
+### Example 2: Get a specific SMB share on a the local computer
```
Comment (get-itips) [inline docset/windows/smbshare/Get-SmbShare.md:72]:
```suggestion
### Example 2: Get a specific SMB share on the local computer
```

File: docset/windows/smbshare/Get-SmbShare.md (hunk 1 above)
Comment (msbemba) [reply docset/windows/smbshare/Get-SmbShare.md]: Thank you. I have added a period at the end of the sentence.

File: docset/winserver2012r2-ps/smbshare/Get-SmbShare.md (hunk 2 above)
Comment (msbemba) [reply docset/winserver2012r2-ps/smbshare/Get-SmbShare.md]: Thank you. I have added a period at the end of the sentence.

**Comment by velkovb** [comment]
This also resolves https://github.com/MicrosoftDocs/windows-powershell-docs/issues/1996

**Pull request merged**